### PR TITLE
Fix #341

### LINKED ISSUE #341: Ambiguous `--help` message in tldr client
## Description
This is what I got when I simply run `tldr`:

```
Usage: tldr command [options]

Simplified and community-driven man pages

Options:
  -v, --version            output the version number
  -l, --list               List all commands for the chosen platform in the cache
  -a, --list-all           List all commands in the cache
  -1, --single-column      List single command per line (use with options -l or -a)
  -r, --random             Show a random command
  -e, --random-example     Show a random example
  -f, --render [file]      Render a specific markdown [file]
  -m, --markdown           Output in markdown format
  -o, --os [type]          Override the operating system [linux, osx, sunos]
  --linux                  Override the operating system with Linux
  --osx                    Override the operating system with OSX
  --sunos                  Override the operating system with SunOS
  -t, --theme [theme]      Color theme (simple, base16, ocean)
  -s, --search [keywords]  Search pages using keywords
  -u, --update             Update the local cache
  -c, --clear-cache        Clear the local cache
  -h, --help               display help for command

  Examples:

    $ tldr tar
    $ tldr du --os=linux
    $ tldr --search "create symbolic link to file"
    $ tldr --list
    $ tldr --list-all
    $ tldr --random
    $ tldr --random-example

  To control the cache:

    $ tldr --update
    $ tldr --clear-cache

  To render a local file (for testing):

    $ tldr --render /path/to/file.md
```

Notice that the description next to the option `-h, --help` was _"display help for command"_. This is a little ambiguous to me, since `command` is already defined as a required program argument. However, it turns out that they are completely two different things.

## Expected behaviour
The description should be _"show this help message"_ or similar things.

## Actual behaviour
It was _"display help for command"_.

## Environment
- Operating system - Windows 10
- NodeJS version: v.14.16.1

### PR DESCRIPTION
**Pull request by davidhcefx**
## Description
Fix #341, which is to remove ambiguous descriptions with clearer ones.
- `-v, --version`: `output the version number` -> `Display version`
- `-h, --help`: `display help for command` -> `Show this help message`

## Checklist

Please review this checklist before submitting a pull request.

- [x] Code compiles correctly
- [x] Created tests, if possible
- [x] All tests passing (`npm run test:all`)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

**Review by agnivade — APPROVED** [review]

**Comment by vladimyr** [comment]
And I was just about to ask to fix bash completion that apparently uses `-V, --version`... 🙃

**Comment by agnivade** [comment]
I was waiting for a review from you. Anyways, we can send another PR for that.

**Comment by vladimyr** [comment]
Sure thing, no worries 👍

**Pull request merged**
